### PR TITLE
Added GitHub Dimmed Theme

### PR DESCRIPTION
**Pull request by MrBlueBird2**
Please approve and merge my pull request, Theme name: GitHub Dimmed Theme

### CODE REVIEW

File: src/theme.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-export const COLORS: {[name: string]: Theme} = {
+export const COLORS: { [name: string]: Theme } = {
```
Comment (ryo-ma) [inline src/theme.ts:1]: Can you remove the spaces?

File: src/theme.ts [hunk 2]
```diff
@@ -407,6 +407,30 @@ export const COLORS: {[name: string]: Theme} = {
     DEFAULT_RANK_SHADOW: "#6272a4",
     DEFAULT_RANK_TEXT: "#0d1117",
   },
+  github: {
```
Comment (ryo-ma) [inline src/theme.ts:410]: There is already a theme called darkhub, what's the difference?

File: src/theme.ts (hunk 1 above)
Comment (MrBlueBird2) [reply src/theme.ts:1]: Yeah. sure, but why??

File: src/theme.ts (hunk 2 above)
Comment (MrBlueBird2) [reply src/theme.ts:410]: This is Github dark dimmed theme, and this is different from that.

File: src/theme.ts (hunk 1 above)
Comment (ryo-ma) [reply src/theme.ts:1]: I misunderstood. There is no problem because it conforms to deno fmt.

Comment (MrBlueBird2) [reply src/theme.ts]: ok

File: src/theme.ts (hunk 2 above)
Comment (ryo-ma) [reply src/theme.ts:410]: OK.  You should be renamed to identify the theme as "GitHub dark dimmed".

Comment (MrBlueBird2) [reply src/theme.ts:410]: Ok

File: src/theme.ts [hunk 3]
```diff
@@ -322,7 +322,8 @@ export const COLORS: {[name: string]: Theme} = {
     SECRET_RANK_3: "#fff",
     SECRET_RANK_TEXT: "#f73155",
     NEXT_RANK_BAR: "#f73155",
-    S_RANK_BASE: "#ff8400",
+    S_RANK_BASE: "#ff8400",heme 
+
```
Comment (ryo-ma) [inline src/theme.ts:326]: Do not change here.
Can you undo it?

Comment (MrBlueBird2) [reply src/theme.ts:326]: Yeah, sure, I changed it by mistake, Sorry!

Comment (MrBlueBird2) [reply src/theme.ts]: Done!

**Pull request merged**